thunderbird-development/building-thunderbird/linux-build-prerequisites.md

```diff
@@ -4,41 +4,97 @@ description: >-
   environment set up and ready to hack on Thunderbird.
 ---
 
-# Linux Build Prerequisites
-
-This guide assumes you already followed the [Getting Started](../getting-started.md) documentation and you already downloaded `mozilla-central` and `comm-central` source code.
+# Hardware Requirements
 
 ## 64-bit version
 
-You need a 64 bit version of Linux in order to build Thunderbird. You can check which version you're running by typing this command in your terminal:
+You will need to be running a 64-bit version of Linux in order to build Thunderbird. You can check which version you're running by typing this command in your terminal:
 
 ```
 uname -m
 ```
 
 if this command returns `x86_64` you can proceed.
 
+## 30 GB of free space
+
+The Thunderbird build can use 30-40GB of disk space to complete depending on your operating system.
```

Comment:
Should we repeat this in every build page or just leave it in the first starting page?

Reply:
Well I had initially left it only on the first starting page, under the hardware requirements. But then it seemed to small to draw attention. If you think it's too redundant to have in each page I can remove it, however I bet people will not really *see* it.

Reply:
Sounds good, we can keep it